docs/src/main/sphinx/object-storage/file-system-azure.md

```diff
@@ -91,3 +91,14 @@ Storage:
 * - `azure.oauth.secret`
   - A OAuth 2.0 client secret for the service principal.
 :::
+
+## Access multiple storage accounts
+
+To allow Trino to access multiple Azure storage accounts from a single
+catalog configuration, you can use [](azure-oauth-authentication) with a
+service principal. First, create a service principal in Azure Active
+Directory (App registrations) and save the client secret. Then, grant the
+service principal access by assigning appropriate roles, such as Storage
+Blob Data Contributor, to each storage account. This is done via the Access
+Control (IAM) section, where you add role assignments and select the service
```

Comment:
Again .. what is "Access Control (IAM)" .. if this is a title or screen or table name in the Ui.. use bold